src/lib/startPodcastFromTime.ts

```diff
@@ -0,0 +1,27 @@
+import AsyncStorage from "@react-native-community/async-storage"
+import { PV } from "../resources"
+
+export const getStartPodcastFromTime = async (podcastId?: string) => {
+  if (podcastId) {
+    const allStartPodcastFromTime = await getAllStartPodcastFromTime()
+    return allStartPodcastFromTime[podcastId] || 0
+  } else {
+    return 0
+  }
+}
+
+export const getAllStartPodcastFromTime = async () => {
+  const limits = await AsyncStorage.getItem(PV.Keys.START_PODCAST_FROM_TIME)
+  return limits ? JSON.parse(limits) : {}
+}
+
+export const setStartPodcastFromTime = async (podcastId?: string, seconds?: number) => {
+  if (podcastId) {
+    const allStartPodcastFromTime: any = await getAllStartPodcastFromTime()
+    allStartPodcastFromTime[podcastId] = seconds && seconds > 0 ? seconds : null
```

Comment:
I think here instead of setting it null, you should delete the property from the object.
```
if (!seconds) {
  delete allStartPodcastFromTime[podcastId]
} else {
  allStartPodcastFromTime[podcastId] = seconds
}
```